src/Components/Firestore.js

```diff
@@ -57,6 +57,8 @@ export async function SaveToFirestore(user, localUser) {
           bio: localUser?.bio ?? null,
           top8: [...localUser.top8],
           reviews: [...localUser.reviews],
+          comments:
```

Comment:
To not break existing users who have localUsers without a comments field before they save, we should probably also make sure that the 'comments' field exists in localUser in PopulateFromFirestore.  I'll add a commit to do that.